v1.1/demo-follow-the-workload.md

```diff
@@ -0,0 +1,298 @@
+**Range Replica** | CockroachDB replicates each range (3 times by default) and stores each replica on a different node.
+**Range Lease** | For each range, one of the replicas holds the "range lease". This replica, referred to as the "leaseholder", is the one that receives and coordinates all read and write requests for the range.
+
+### How It Works
+
+"Follow the workload" is based on the way **range leases** handle read requests. Read requests bypass the Raft consensus protocol, accessesing the range replica with which the range lease is co-located (the leaseholder) and sending the results to the client without needing to coordinate with any of the other range replicas. Bypassing Raft, and the network roundtrips involved, is possible because the leaseholder is guaranteed to be up-to-date already due to the way write request are handled by Raft.
+
+This increases the speed of reads, but it doesn't guarantee that the range lease will be anywhere close to the origin of requests. If requests are coming from the US West, for example, and the relevant range lease is on a node in the US East, the requests would likely enter a gateway node in the US West and then get routed to the node with the range lease in the US East.
+
+However, you can cause the cluster to actively move range leases for even better read performance by starting each node with the `--locality` flag. With this flag specified, the cluster knows about the location of each node, so when there's high latency between nodes, the cluster will move active range leases to a node closer to the origin of the majority of the workload. This is especially helpful for applications with workloads that move around throughout the day (e.g., most of the traffic is in the US East in the morning and in the US West in the evening).
+
+### Example
+
+In this example, let's imagine that lots of read requests are going to node 1, and that the requests are for data in range 3. Because range 3's lease is on node 3, the requests are routed to node 3, which returns the results to node 1. Node 1 then responds to the clients.
+
+<img src="{{ 'images/follow-workload-1.png' | relative_url }}" alt="Follow the workload example" style="max-width:100%" />
+
+However, if the nodes were started with the `--locality` flag, after a short while, the cluster would move range 3's lease to node 1, which is closer to the origin of the workload, thus reducing the network roundtrips and increasing the speed of reads.
+
+<img src="{{ 'images/follow-workload-2.png' | relative_url }}" alt="Follow the workload example" style="max-width:100%" />
+
+## Tutorial
+
+### Step 1. Install prerequisites
+
+In this tutorial, you'll use CockroachDB, a Go network tool to simulate network latency on your local workstation, and a Go load generator to simulate client workloads. Before you begin, make sure these applications are installed:
```

Comment:
Nit: Can we phrase this as "In this tutorial, you'll use CockroachDB, comcast (a Go network tool....)..." 

The way it reads now seems odd - as if CockroachDB is the Go network tool :)